package.json

```diff
@@ -21,7 +21,9 @@
     "get-console-version": "doc-tools get-console-version",
     "build": "antora --to-dir docs --fetch local-antora-playbook.yml",
     "serve": "wds --node-resolve --open preview/test/ --watch --root-dir docs",
-    "test": "jest"
+    "test": "jest",
```

Comment:
Please bump the version in this file. Since this is quite a large change, bump the minor version to 4.8.0 and then `npm install` to update the lock file.